contracts/ArrowToken.sol

```diff
@@ -1,27 +1,36 @@
+// SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
-// SPDX-License-Identifier: MIT
+import "@openzeppelin/contracts-upgradeable/token/ERC20/ERC20Upgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
 
-import "@openzeppelin/contracts/access/Ownable.sol";
-import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+/// @title Arrow Token implementation that is upgradeable
+/// @notice Create ARROW tokens of an `initialSupply` amount
+/// @dev If we were to upgrade the initializer by introducing new variables, we need to create a normal function like `initializeV2` (https://forum.openzeppelin.com/t/uups-proxies-tutorial-solidity-javascript/7786/53), and make sure to include all existing variables from the current version of the contract to avoid storage collision
```

Comment:
Please follow the existing style for comments, so we're not mixing different styles in the codebase.
```suggestion
/**
    @title Arrow Token implementation that is upgradeable
    @notice Create ARROW tokens of an `initialSupply` amount
    @dev If we were to upgrade the initializer by introducing new variables, we need to create a normal function like `initializeV2` (https://forum.openzeppelin.com/t/uups-proxies-tutorial-solidity-javascript/7786/53), and make sure to include all existing variables from the current version of the contract to avoid storage collision
*/
```